node/light_client.go

```diff
@@ -0,0 +1,181 @@
+package node
+
+import (
+	"context"
+
+	"github.com/tendermint/tendermint/config"
+	tmbytes "github.com/tendermint/tendermint/libs/bytes"
+	rpcclient "github.com/tendermint/tendermint/rpc/client"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+	"github.com/tendermint/tendermint/types"
+)
+
+var _ rpcclient.Client = &LightClient{}
+
+type LightClient struct {
+	*types.EventBus
+	config *config.RPCConfig
+
+	node LightNode
+}
+
+func NewLightClient(node *LightNode) *LightClient {
+	panic("Not implemented")
```

Comment:
This is something we actually should implement.

Reply:
done